### PR TITLE
Newbranch

### PR DESCRIPTION
**Pull request by madhumitha13579**

### CODE REVIEW

File: src/app/dashboard/dashboard.component.ts [hunk 1]
```diff
@@ -50,38 +42,18 @@ export class DashboardComponent implements OnInit {
           showeditbutton:true
         }
       });
-    dialogRef.afterClosed().subscribe((t: any) => { console.log('output', `${t}`) })
+    dialogRef.afterClosed().subscribe((details: any) => ( `output, ${details}`) )
```
Comment (Sushmitha-Vk) [inline src/app/dashboard/dashboard.component.ts]: what is this output...? was it not part of console.log...?

File: src/app/header/header.component.ts [hunk 2]
```diff
@@ -30,30 +30,22 @@ localhost: any;
   ngOnInit(){
     this.a =localStorage.getItem('email')
```
Comment (Sushmitha-Vk) [inline src/app/header/header.component.ts]: why a..? why not changed to proper name?

File: src/app/header/header.component.ts [hunk 3]
```diff
@@ -30,30 +30,22 @@ localhost: any;
...
     this.name=this.a.split('.').join(' ').split('@',1).join(' ')
-    // console.log(this.name)
     this.f=((this.name.split(' ',1))[0])[0]  
     this.l=((this.name.split(' ',2))[1])[0]
```
Comment (Sushmitha-Vk) [inline src/app/header/header.component.ts]: same here

File: src/app/login/login.component.ts [hunk 4]
```diff
@@ -16,31 +16,18 @@ userdetail:any
...
+      email:new FormControl('', Validators.required),
+      password:new FormControl('', Validators.required )
+    })
   }
   openSnackBar() {
     this._snackBar.open('Successful')
   }
   submit(data:any){
-    //console.log('..', data)
     this.service.login(data).subscribe((d:any)=>{
```
Comment (Sushmitha-Vk) [inline src/app/login/login.component.ts]: d naming

File: src/app/stepper/stepper.component.ts [hunk 5]
```diff
@@ -28,10 +28,10 @@ import { MatDialog, MAT_DIALOG_DATA } from '@angular/material/dialog';
...
-  
-states:any
-  
-firstFormGroup = this._formBuilder.group({
+
+  states: any
+
+  firstFormGroup = this._formBuilder.group({
     name: ['', Validators.required],
     //email:['', Validators.required],
```
Comment (Sushmitha-Vk) [inline src/app/stepper/stepper.component.ts:36]: remove comments

File: src/app/wishlist/wishlist.component.ts [hunk 6]
```diff
@@ -11,59 +11,45 @@ import { SampleserviceService } from '../sampleservice.service';
...
-ngOnInit() {
-  this.serv.getCustomer()
-this.dataSource=this.serv.dataEvent$.pipe(map((n:any)=>{
-  // console.log("..................", n)
-  return n.filter((x:any)=>x.Wish)
-  
-}))
+  ngOnInit() {
+    this.serv.getCustomer()
+    this.dataSource = this.serv.dataEvent$.pipe(map((n: any) => {
```
Comment (Sushmitha-Vk) [inline src/app/wishlist/wishlist.component.ts]: n..?

File: src/app/wishlist/wishlist.component.ts [hunk 7]
```diff
@@ -11,59 +11,45 @@ import { SampleserviceService } from '../sampleservice.service';
...
+  }
+  edit(element: any) {
+    const dialogRef = this.dialog.open(AddcustomerComponent,
+      {
+        data: {
+          ...element,
+          showeditbutton: true
+        }
+      });
+    dialogRef.afterClosed().subscribe((details: any) => { })
```
Comment (Sushmitha-Vk) [inline src/app/wishlist/wishlist.component.ts]: details not used, why is it declared..?

**Review by Sushmitha-Vk — CHANGES_REQUESTED** [review]

File: src/app/dashboard/dashboard.component.ts (hunk 1 above)
Comment (madhumitha13579) [reply src/app/dashboard/dashboard.component.ts]: changes corrected

File: src/app/header/header.component.ts (hunk 2 above)
Comment (madhumitha13579) [reply src/app/header/header.component.ts]: changes corrected

File: src/app/header/header.component.ts (hunk 3 above)
Comment (madhumitha13579) [reply src/app/header/header.component.ts]: changes corrected

File: src/app/login/login.component.ts (hunk 4 above)
Comment (madhumitha13579) [reply src/app/login/login.component.ts]: changes corrected

File: src/app/stepper/stepper.component.ts (hunk 5 above)
Comment (madhumitha13579) [reply src/app/stepper/stepper.component.ts:36]: corrected mam

File: src/app/wishlist/wishlist.component.ts (hunk 6 above)
Comment (madhumitha13579) [reply src/app/wishlist/wishlist.component.ts]: corrected changes

**Review by Sushmitha-Vk — APPROVED** [review]

**Pull request merged**